### ISSUE TITLE
Weird borders in some browsers from missing src attribute.

### ISSUE THREAD

**Issue by inbetweencode** [kylereicks/picturefill.js.wp#40]
I'm not very happy with with the weird unremovable borders that appear (alongside the alt text) because of the initial missing (mandatory) src attribute. That isn't pretty on a production website.

I'm thinking that it might be a good idea (in my current case at least) to insert an initial src attribute with some base64 encoded transparent gif.

Do you have an opinion about that? Do you have an insight which filter I could use best?
